src/lib/core/error/error-options.ts

```diff
@@ -7,27 +7,24 @@
  */
 
 import {InjectionToken} from '@angular/core';
-import {FormControl, FormGroupDirective, NgForm} from '@angular/forms';
+import {FormGroupDirective, NgForm, NgControl} from '@angular/forms';
 
 /** Injection token that can be used to specify the global error options. */
 export const MD_ERROR_GLOBAL_OPTIONS = new InjectionToken<ErrorOptions>('md-error-global-options');
 
 export type ErrorStateMatcher =
```

Comment:
@crisbeto I added it only because it seemed to be a common question here and on SO https://github.com/angular/material2/pull/4750#issuecomment-307525083, https://github.com/angular/material2/pull/4750#discussion_r123363437

@mmalerba None that I was aware of. Simply based it off of `PlaceholderOptions` which seemed a nicely future proof